### DISCUSSION TITLE
How can i reduce the image size limit on temporary upload

### DISCUSSION THREAD

**Discussion by Timbaron** [spatie/laravel-medialibrary#2663] [General] [answered]
![Screenshot from 2021-11-03 17-35-23](https://user-images.githubusercontent.com/36372628/140103883-28bf9528-44c5-4bf3-b4e5-0bf19d1146a2.png)

I'd like to reduce the image size limit on the teporay upload side. 

E.g if image is larger than 2mb it should not upload on the UI at all. I'm trying to reduce the size limit(from the image above) from 256 to 2mb.

Any ideas on how i can achieve that?

**Comment by erikn69** [comment, marked as answer]
https://github.com/spatie/laravel-medialibrary/blob/a7e961d75ffc0547870c23c2e9a8cd334f2a20a4/config/media-library.php#L15

**Reply by Timbaron** [reply]
![image](https://user-images.githubusercontent.com/36372628/140293808-73226b3c-7111-4984-a7f6-3142a416e851.png)
I already changed that, This means 50kb right, But if i upload images of 2mb it accepts them in the temporary upload but if i submit my form for overall upload it displays the error below. 
**Spatie\MediaLibrary\MediaCollections\Exceptions\FileIsTooBig
File `/tmp/media-library0yMJYv` has a size of 89.35 KB which is greater than the maximum allowed 51.2 KB** 

We is something i don't want to show my users on production. What i want to know is if it's possible to not allow the admin to be able to upload images above 50kb on the temporary upload.

**Reply by erikn69** [reply]
For that you have to change the config of your php.ini
`upload_max_filesize` 
`post_max_size`
